blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,77 @@
+title: With love, Team Bundledore, India !
+ layout: post
+ created_at: Saturday August 14 2017
+ permalink: blog/2017-08-14-bundledore-with-love-team-Bundledore-India
+ current: blog
+ author: Bundledore
+ twitter: TeamBundledore
+ categories:student-posts
+ 
+ 
+ ### Hello, people! 
+ A very Happy Independence Day to all my fellow Indians here :) We are Team Bundledore, and our project is [AFDC League Management System](https://leagues.afdc.com/) 
```

Comment:
This is kind of a controversial line as independence day of India is 15th August, 2017. Either drop it or you can change the date details to 15th August. Anyway we are slightly behind the schedule so I can publish this on 15th. :)